index.html

```diff
@@ -8,61 +8,98 @@
     <title>Company Site</title>
     <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW" />
   </head>
+
```

Comment:
Something I've been doing lately is if there is an closing tag immediately followed by an opening tag and they are sibling elements, add a space between them! Makes it way easier to see where tags begin and end!